site/build_docs.py

```diff
@@ -137,11 +140,15 @@ def run_hugo(
             git_checkout(tag.name, temp_repo, Path(temp_dir))
             change_version_menu_toml(versioning_toml_path, tag.name)
             run_npm_install()
+
+            # find correct hugo version
+            hugo = hugo110 if version.parse(tag.name) >= UPDATED_HUGO_FROM else hugo83
```

Comment:
I thought that this naming pattern may break at some point, but now it seems fine.